Formula/linthesia.rb

```diff
@@ -0,0 +1,28 @@
+class Linthesia < Formula
+  desc "Synthesia-like software for Linux"
+  homepage "http://linthesia.sourceforge.net/"
+  url "https://github.com/linthesia/linthesia/archive/refs/tags/0.8.0.tar.gz"
+  sha256 "0bbfeb30909fbd3a3b33bc45e8f7e75944ba6b2135a627d5729f4bda50540027"
+  license "GPL-2.0-only"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "gsettings-desktop-schemas" => :build
+
+  def install
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+  end
+
+  def post_install
+    system Formula["glib"].opt_bin/"glib-compile-schemas", HOMEBREW_PREFIX/"share/glib-2.0/schemas"
+    system Formula["gtk+3"].opt_bin/"gtk3-update-icon-cache", "-qtf", HOMEBREW_PREFIX/"share/icons/hicolor"
+  end
+
+  test do
+    system bin/"linthesia", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
This needs a midi file and will popup a GUI.

Is that ok?

Reply:
That's fine. Alternatively, we could try doing something with a non-existent midi file (or a non-midi file) and check whether it generates the expected error.